h2o-py/h2o/__init__.py

```diff
@@ -61,12 +61,31 @@ def readTxtFromWhl(name, fallback):
 if (__version__.endswith("99999")):
     print(__buildinfo__)
 
-__all__ = ("connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
+
+__all__ = ["connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
            "import_sql_select", "parse_setup", "parse_raw", "assign", "deep_copy", "get_model", "get_grid", "get_frame",
            "show_progress", "no_progress", "enable_expr_optimizations", "is_expr_optimizations_enabled", "log_and_echo",
            "remove", "remove_all", "rapids", "ls", "frame", "import_hive_table",
            "frames", "download_pojo", "download_csv", "download_all_logs", "save_model", "download_model", "upload_model", "load_model", "export_file",
            "cluster_status", "cluster_info", "shutdown", "create_frame", "interaction", "as_list", "network_test",
-           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset","flow",
+           "set_timezone", "get_timezone", "list_timezones", "demo", "make_metrics", "cluster", "load_dataset", "flow",
            "upload_custom_metric", "upload_custom_distribution",  "mojo_predict_csv", "mojo_predict_pandas", "import_mojo", 
-           "upload_mojo", "print_mojo", "load_grid", "save_grid")
+           "upload_mojo", "print_mojo", "load_grid", "save_grid"]
+
+try:
+    # Export explain functions that are useful for lists of models
+    from h2o.explain import explain, explain_row, variable_importance_heatmap, model_correlation_heatmap, \
+        partial_dependences, register_explain_methods
+
+    __all__ += [
+        "explain",
+        "explain_row",
+        "variable_importance_heatmap",
```

Comment:
I don't have a strong opinion at this point but thinking from the top  of my mind,
do we need to name function based on the viz it's generating rather than just a `type` param?
For e.g. `variable_importance(type='heatmap', ...)`. 
That way if a single model is passed then it generates a `bar` plot and if a list of models is passed then it generates a heatmap.

If it's too much effort right now, we make modifications for future releases.

Reply:
I like the idea, that way we could also do a box plot for multiple models with the same API etc.

I would probably keep it the way it is right now (not much time before release) and in future make a function/method `variable_importance` that would dispatch depending on the input if that sounds reasonably.

Reply:
sounds good.

Reply:
if `variable_importance_heatmap` is the only one supported currently, you can still rename it to `variable_importance` and implement the dispatch later, right? This would prevent you from having to implement deprecations later..